test/e2e/kubectl.go

```diff
@@ -188,6 +195,114 @@ var _ = Describe("Kubectl client", func() {
 			if e, a := "hi", execOutput; e != a {
 				Failf("Unexpected kubectl exec output. Wanted %q, got %q", e, a)
 			}
+
+			By("executing a command in the container through an HTTP proxy")
```

Comment:
I'm wondering if it would make more sense for this to be an `It()` sibling to `It("should support exec"`